LICENSE.txt

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2015, Jesse Berger (user alias: valadian)
```

Comment:
If we were to do a rewrite, would we be able to modify the license if we didn't use any significant portions of the original authors code? (Not that we're going to rewrite modcore though, lol)